Changelog.md

```
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## v0.0.2-rc142
 
 ### Changed
+  - feat/555-add-compacting-middleware (2026-01-31)
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)